### PR TITLE
feat: add a new feature in main section

### PR DESCRIPTION
**Pull request by MuntasirAnik**
# Description

A component is created  to add a new Feature as per ticket #82. 

Fixes # (issue)

   1. New Feature Added as Advance Section.
  
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code

2023-01-09 at 9 12 07 PM

### CODE REVIEW

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
looks good! do the chagnes and repush for approval

File: apps/akij-food-beverage/src/components/ui/features/AdvanceSection/Advance.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import {useEffect, useState} from 'react'
+import { fetchProducts } from '../../../../utils/fetchProducts';
+import { urlForThumbnail } from './../../../../utils/imageProcess'
+
+interface Product {
```
[the remark this answers is not in the document]
Comment (MuntasirAnik) [reply apps/akij-food-beverage/src/components/ui/features/AdvanceSection/Advance.tsx]: Added in types

**Comment by urmirahman** [comment]
@TanvirAlam  what happened here???

**Comment by TanvirAlam** [comment]
@urmirahman build failed, see why

**Review by TanvirAlam — APPROVED** [review]
LGTM

**Pull request merged**